Intersections_2/test/Intersections_2/test_intersections_2.cpp

```diff
@@ -1,13 +1,9 @@
 // 2D intersection tests.
 
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Homogeneous.h>
+
 #include <CGAL/Intersection_traits_2.h>
-#include <CGAL/Object.h>
-#include <CGAL/Line_2.h>
-#include <CGAL/Point_2.h>
-#include <CGAL/Segment_2.h>
-#include <CGAL/Ray_2.h>
-#include <CGAL/Triangle_2.h>
-#include <CGAL/Iso_rectangle_2.h>
```

Comment:
I disagree with that. In the approach include-what-you-use, as `CGAL::Point_2`, `CGAL::Triangle_2`,... are used directly, as opposed to usages of `Kernel::Point_2`¸ I think the corresponding headers must be included. That is probably what the analyzing tool [include-what-you-use] would recommend.

[include-what-you-use]: https://github.com/include-what-you-use/include-what-you-use

Reply:
Let me then rewrite it to `Kernel::Point_2` etc.

Reply:
Why not.

But I think there should be a few tests, in CGAL, that keep using the global names `CGAL::Point_2`, `CGAL::Segment_2`...

Reply:
What do you suggest to test?

Reply:
I do not know. Just test that the code that uses `CGAL::Obj<K>` instead of `K::Obj` works as intended. If it happens we break by accident the type equality between `K::Obj` and `CGAL::Obj<K>`, the testsuite should fail.